src/shiftedNormL1Binf.jl

```diff
@@ -27,7 +27,7 @@ mutable struct ShiftedNormL1BInf{
   end
 end
 
-(ψ::ShiftedNormL1BInf)(y) = ψ.h(ψ.xk + ψ.sj + y) + IndBallLinf(ψ.Δ)(ψ.sj + y)
+(ψ::ShiftedNormL1BInf)(y) = ψ.h(ψ.xk + ψ.sj + y) + (ψ.χ(ψ.sj + y) ≈ ψ.Δ ? IndBallLinf(ψ.Δ)([ψ.Δ]) :  IndBallLinf(ψ.Δ)(ψ.sj + y))
```

Comment:
what does this change?

Reply:
I was getting an error where \psi.sj + y was 1e-16 outside of the Binf ball. This fixes that by replacing ψ.χ(ψ.sj + y)  with ψ.Δ if they are very close, getting rid of the Inf otherwise.

Reply:
so basically the same as https://github.com/JuliaSmoothOptimizers/ShiftedProximalOperators.jl/blob/master/src/shiftedIndBallL0BInf.jl#L41 ?

Reply:
Yes it fixes a similar problem.